Makefile

```diff
@@ -58,6 +58,9 @@ TEMPLATES := $(foreach C,$(COHORTS),templates/$(C).tsv)
 # OWL file in the build directory for all cohorts (contains xrefs)
 ONTS := $(foreach C,$(COHORTS),build/$(C).owl)
 
+xx:
+	echo $(COHORTS)
```

Comment:
I would use `make -pn | grep COHORTS`

Reply:
the xx task should not be there at all. Mistake in haste (just needed during dev). Thanks for tip!